amlip_docs/index.rst

```diff
@@ -76,6 +74,8 @@
    /rst/developer_manual/installation/sources/linux/linux
    /rst/developer_manual/installation/sources/windows/windows
    /rst/developer_manual/installation/configuration/cmake_options
+   /rst/developer_manual/technologies/technologies
```

Comment:
“The best argument against democracy is a five-minute conversation with the average voter (Winston Churchill)".